### ISSUE TITLE
suppress duplicate listing of People tagged as Favourites or Low Priority

### ISSUE THREAD

**Issue by jonassmedegaard** [quotient-im/Quaternion#661]
In Room list, people are still listed in section "People" when tagged as either Favourites or Low Priority.

Please suppress their listing in section "People" in those cases, so that the full listing does not _grow_ when doing tagging that I can only interpret as _prioritization_ hints.

**Comment by KitsuneRal** [comment]
The way Quaternion handles both tags and direct chat flags as of now is uniform; simply put, each group lists all rooms marked with a given tag. Unlike Riot, Quaternion allows you to mark a room with several tags at once, supporting cases, e.g., when a certain room is marked with a custom "IRC" tag and at the same time is in "Favourites". The issue specifically with direct chats is that Quaternion doesn't (yet) have a means to unset the direct chat flag (#552) so you have no control over the "People" group.

With all that said - apparently, you do not expect that groups can overlap so I have a couple of questions in order to figure out what's best to do:
1. Do you use custom tags? If not, I might just provide some configuration switch that enables the behaviour similar to Riot.
2. If you use custom tags - assuming that #552 is fixed, do you expect the same behaviour across all groups (as per example above, a room would be displayed either in "IRC" or in "Favourites" but not both) or is this expectation specific to "People"?

**Comment by jonassmedegaard** [comment]
No, I don't use custom tags, but I might (and think that is ortogonal to 
my expectations here: My expectation is to *somehow* be able to not 
duplicate room links but _move_ room links from one section to another.

How about an option to "hide duplicates"?: When enabled (from an item in 
the "Rooms" menu), rooms included in several sections will appear only 
in the topmost one and subsequent sections will indicate how many rooms 
were suppressed.

E.g. (with 3 people, one of them also tagged as favourite):

Favourites (1 room(s))
* bar

People (3 room(s), 2 shown)
* foo
* baz

Ungrouped rooms (4 room(s), 1 shown)
* poof


-- 
 * Jonas Smedegaard - idealist & Internet-arkitekt
 * Tlf.: +45 40843136  Website: http://dr.jones.dk/

 [x] quote me freely  [ ] ask before reusing  [ ] keep private

**Comment by KitsuneRal** [comment]
Ok, thanks for input.
